config/installers/rolebasedcreds/namespaced/kustomization.yaml

```diff
@@ -57,6 +57,18 @@ patchesJson6902:
     - op: replace
       path: "/subjects/0/name"
       value: sagemaker-k8s-operator-default
+# Hacky solution so that its possible to add --namespace for both China and US region with this one file
+# change the position of pods so that manager pod is first pod in both US and China
```

Comment:
Sorry I don't quite follow. Why is this necessary?

Reply:
manager pod is at index 1 in US and index 0 in China 

path: "/spec/template/spec/containers/0/args/-"
doesn't allow us to specify last item is an array with index -1 

moving the manager pod so that it is at index 0 in both US and China